test/check-application

```diff
@@ -636,15 +636,24 @@ class TestApplication(testlib.MachineCase):
         self.login_and_go("/podman", authorized=False, superuser=False)
         b.click("#app .pf-c-empty-state button.pf-m-primary")
 
-        b.wait_present("#containers-containers")
-        b.wait_not_present("#overview div.pf-c-alert")
+        # HACK: https://github.com/containers/libpod/issues/6660
+        # Try to click on the button bunch of times due to the aforementioned bug
+        with b.wait_timeout(8):
+            for _ in range(5):
+                try:
+                    b.wait_present("#containers-containers")
+                    b.wait_not_present("#overview div.pf-c-alert")
+                except testlib.Error:
+                    b.click("#app .pf-c-empty-state button.pf-m-primary")
```

Comment:
This needs an `else: <fail>` thing, otherwise it will silently pass  after 5 failures in a row. Also, supposedly you want to add a `break` after the wait_not_present for the success case?

Reply:
> otherwise it will silently pass after 5 failures in a row.

not really, the following commands would fail.

> supposedly you want to add a break after the wait_not_present for the success case?

of course, fixed.